packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/representations/RepresentationService.java

```diff
@@ -50,6 +51,12 @@
 @Service
 public class RepresentationService implements IRepresentationService, IRepresentationPersistenceService, IDanglingRepresentationDeletionService {
 
+    public static final String EXPLORER_DESCRIPTION_ID = UUID.nameUUIDFromBytes("explorer_tree_description".getBytes()).toString();
+
+    public static final String EXPLORER_DOCUMENT_KIND = "siriusComponents://representation?type=Tree";
+
+    public static final String EXPLORER_NAME = "Explorer";
```

Comment:
It would be better to have those contants in the class providing the explorer instead of copying them here since they will change over time.